freeipa.spec.in

```diff
@@ -1323,6 +1331,45 @@ fi
 # with_selinux
 %endif
 
+%triggerin client -- sssd-common < 2.10
+# Has the client been configured?
+restore=0
+test -f '/var/lib/ipa-client/sysrestore/sysrestore.index' && restore=$(wc -l '/var/lib/ipa-client/sysrestore/sysrestore.index' | awk '{print $1}')
+
+SSH_CLIENT_SYSTEM_CONF="/etc/ssh/ssh_config"
+if [ -f "$SSH_CLIENT_SYSTEM_CONF" ]; then
+    # https://pagure.io/freeipa/issue/9536
+    # downgrade sss_ssh_knownhosts with sss_ssh_knownhostsproxy
+    if [ -f '/usr/bin/sss_ssh_knownhosts' ]; then
+        if grep -E -q 'Include' $SSH_CLIENT_SYSTEM_CONF  2>/dev/null ; then
+            SSH_CLIENT_SYSTEM_CONF="/etc/ssh/ssh_config.d/04-ipa.conf"
+        fi
+        GLOBALKNOWNHOSTFILE="GlobalKnownHostsFile /var/lib/sss/pubconf/known_hosts/"
+        grep -qF ($GLOBALKNOWNHOSTFILE) $SSH_CLIENT_SYSTEM_CONF
```

Comment:
If the client is downgraded from sssd 2.10 to 2.9, I see the following error message:
```
  Downgrading      : sssd-common-2.9.4-1.fc39.x86_64                                                                                                                       10/42 
  Running scriptlet: sssd-common-2.9.4-1.fc39.x86_64                                                                                                                       10/42 
/var/tmp/rpm-tmp.BE0Bdg: line 14: syntax error near unexpected token `('
/var/tmp/rpm-tmp.BE0Bdg: line 14: `        grep -qF ($GLOBALKNOWNHOSTFILE) $SSH_CLIENT_SYSTEM_CONF'
warning: %triggerin(freeipa-client-4.12.0.dev-0.fc39.x86_64) scriptlet failed, exit status 2

Error in <unknown> scriptlet in rpm package sssd-common
```

and the file /etc/ssh/ssh_config.d/04-ipa.conf still contains `KnownHostsCommand /usr/bin/sss_ssh_knownhosts %H` although the command is not available.

Reply:
thanks @flo-renaud , fixed.